doc/man3/OSSL_PARAM.pod

```diff
@@ -71,6 +71,13 @@ is NULL.  The usual full terminating template is:
 
 This can also be specified using L<OSSL_PARAM_END(3)>.
 
+=head2 Functional support
+
+Libcrypto offers a limited set of helper functions to handle
+B<OSSL_PARAM> items and arrays, please see L<OSSL_PARAM_get_int(3)>.
+Applications, providers, etc are free to extend or replace those as
```

Comment:
`etc.` ?

Reply:
`, etc., `  Although it raises the question "what else?"  How about "Developers are free ..."

Also the intro comment says "Fixed" not "Fixes"

I will close my PR.

Reply:
I'll switch to your proposal, @richsalz.  Seems much simpler, yeah